saltstack/base/salt/mount/init.sls

```diff
@@ -0,0 +1,19 @@
+{% if grains['os'] == 'Amazon' %}
```

Comment:
we don't use amazon linux anymore, are you sure this will be amazon in case of centos?

Reply:
I am not, and I do not see any better way to check if the cloud platform is AWS or not. I should check whether the issue affects Azure VMs too, if it does, I could kill two birds with the same stone